docs/mongocli/command/mongocli-atlas-dbusers-create.txt

```diff
@@ -96,7 +98,7 @@ Options
    * - --x509Type
      - string
      - false
-     - X.509 method by which the provided username is authenticated.  Valid values are NONE, MANAGED, or CUSTOMER. This value defaults to "NONE".
+     - X.509 method by which the provided username is authenticated.  Valid values are NONE, MANAGED, or CUSTOMER. If you set this to MANAGED the user authenticates with an Atlas-managed X.509 certificate. If you set this to CUSTOMER, the user authenticates with a self-managed X.509 certificate. If you set this to MANAGED or CUSTOMER, you can't set --awsIAMType or --ldapType to any value other than NONE. This value defaults to "NONE".
```

Comment:
Same as above. Also, please see if you can move the "This value defaults to "None" higher in the text, before the clauses.

Reply:
The default sentence is added to the end of the flag programmatically, so we can't change the placement for that without changing the template